cyences_app_for_splunk/default/data/ui/views/cs_linux_reports.xml

```diff
@@ -170,29 +164,128 @@
   </row>
   <row>
     <panel>
-      <title>User List</title>
-      <input type="text" token="tkn_user_list_USERNAME">
-        <label>USERNAME</label>
+      <title>Linux Group Added/Updated/Removed</title>
+      <input type="text" token="tkn_group">
+        <label>Group</label>
+        <default>*</default>
+        <initialValue>*</initialValue>
+      </input>
+      <input type="multiselect" token="tkn_actions">
+        <label>Action</label>
+        <choice value="*">All</choice>
+        <choice value="*added*">Added</choice>
+        <choice value="*removed*">Removed</choice>
+        <choice value="*updated*">Updated</choice>
+        <delimiter>,</delimiter>
+        <default>*</default>
+        <initialValue>*</initialValue>
+      </input>
+      <table>
+        <search>
+          <query>| inputlookup cs_linux_groups
+| stats latest(*) as * latest(_time) as _time by host group_name
+| search status IN ($tkn_actions$) host=$tkn_host|s$  group_name="*$tkn_group$*"
```

Comment:
Dashboard panels keep waiting for input
![image](https://user-images.githubusercontent.com/20964366/216396358-12e97eea-0bc3-4071-9e2a-16a78f1300d0.png)

http://10.33.1.41:8000/en-US/app/cyences_app_for_splunk/test

Reply:
Fixed the issue. Thanks.